### ISSUE TITLE
richtext: accessibility

### ISSUE THREAD

**Issue by nicopap** [nicopap/cuicui#20]
With complex text transforms that split text into several subsections, we are messing up assistive tech such as readers. It should be possible to put the full text sentence into a component the assistive technology can read.
